Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1266,7 +1264,11 @@ UpdateFrameBufferInfo (
   IN  FRAME_BUFFER_INFO  *FrameBufferInfo
   )
 {
-  FrameBufferInfo->LinearFrameBuffer = PciRead32 (PCI_LIB_ADDRESS (0, 2, 0, 0x18)) & 0xFFFFFF00;
+  UINT32                    FrameBufferAddress;
+
+  FrameBufferAddress = GetDeviceAddr (OsBootDeviceVga, 0);
```

Comment:
we don't need to do this for GFX device.  It is already in platform specific library.  The old code should be fine.